### DISCUSSION TITLE
Custom grid (list of objects) in Pimcore admin panel

### DISCUSSION THREAD

**Discussion by twin-elements** [pimcore/pimcore#16055] [Q&A / Support]
Is it possible to create a custom grid view for some folders in the Pimcore panel? 
For example, for the **/comments** folder.

**Comment by dvesh3** [comment]
please see [Documentation](https://pimcore.com/docs/pimcore/10.6/User_Documentation/DataObjects/Grid_Configuration_General.html#page_Grid-Configuration-CSV-Export)
